python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -127,6 +127,20 @@ $defs:
               type: bool
               default: false
               description: Enable PTP.
+            role:
+              type: str
+              default: "bmca"
+              valid_values:
+                - "bmca"
+                - "master"
+            profile:
+              type: str
+              default: "aes67-r16-2016"
+              description: |-
+                Default available profiles are:
+                  - "aes67"
+                  - "aes67-r16-2016"
+                  - "smpte2059-2"
```

Comment:
```suggestion
              default: bmca
              valid_values:
                - bmca
                - master
                - bmca
            profile:
              type: str
              default: aes67-r16-2016
              description: |-
                Default available profiles are:
                  - "aes67"
                  - "aes67-r16-2016"
                  - "smpte2059-2"
```

Reply:
also we are setting default profile as aes67-r16-2016 then in description mentioning other default profiles too. It will be confusing.

Reply:
Yes, it mean we can apply default available profiles without creating ptp profile. If we are not applying any profile it will by default assign profile `aes67-r16-2016` among default profiles.

Reply:
If any profile other than those 3 default profiles is applied then user should defined the profile.